src/modules/hardware-wallet/src/atomByteOffsetMetadata.ts

```diff
@@ -0,0 +1,85 @@
+import cbor from 'cbor'
```

Comment:
This is for parsing the atom CBOR before signing. The ledger app needs some specific metadata, which is generated here.